.github/workflows/tests.yaml

```diff
@@ -202,6 +203,7 @@ jobs:
                   ./scripts/run_in_build_env.sh \
                   "./scripts/tests/run_test_suite.py \
                      --chip-tool ./out/linux-x64-chip-tool${CHIP_TOOL_VARIANT}-${BUILD_VARIANT}/chip-tool \
+                     --run-yamltests-with-chip-repl \
```

Comment:
Maybe we should add a separate task for running test against `run-yamltests-with-chip-repl` ? 
We have already timed out a few times on running tests with `chip-tool` since there are a lot of tests doing various different things, hence the `65` minutes.
Additional running tests with `chip-repl` is cheap at the moment since it is only running against `TestConstraints.yaml` but it will takes longer and longer...

Additional, and as Boris pointed out this is a bit unclear that this command launch tests against 2 runners. Duplicating it would makes it easier to see.

Reply:
I think it should be the latter, an other test step.

> While I see what you are saying about how this will potentially become too long, we do plan eventually entire replacing running the accessory based yaml tests from chip-tool to python. Once we are ready to actually run all the tests we could make that switch at that time.

Then we would remove the "old" test step once we are ready ;)